samples/snippets/bqml_getting_started_test.py

```diff
@@ -72,8 +67,8 @@ def test_bqml_getting_started(random_model_id):
     # to use as training data.
     features = bpd.DataFrame(
         {
-            "os": operatingSystem,
-            "is_mobile": isMobile,
+            "os": operating_system,
+            "isMobile": is_mobile,
```

Comment:
The `"is_mobile"` string didn't need to be changed, but if you do you must change it everywhere.